metricbeat/module/kafka/_meta/run.sh

```diff
@@ -68,5 +71,18 @@ ${KAFKA_HOME}/bin/kafka-acls.sh --authorizer-properties zookeeper.connect=localh
 
 touch /tmp/.acls_loaded
 
+
+# Start a forever producer
+cat /dev/urandom | KAFKA_OPTS="-Djava.security.auth.login.config=/kafka/bin/jaas-kafka-client-producer.conf -javaagent:/opt/jolokia-jvm-1.5.0-agent.jar=port=8775,host=0.0.0.0" \
+ ${KAFKA_HOME}/bin/kafka-console-producer.sh --topic test --broker-list localhost:9091 --producer.config ${KAFKA_HOME}/bin/sasl-producer.properties > /dev/null &
+
+wait_for_port 8775
+
+# Start a forever consumer
+KAFKA_OPTS="-Djava.security.auth.login.config=/kafka/bin/jaas-kafka-client-consumer.conf -javaagent:/opt/jolokia-jvm-1.5.0-agent.jar=port=8774,host=0.0.0.0" \
+ ${KAFKA_HOME}/bin/kafka-console-consumer.sh --topic=test --bootstrap-server=localhost:9091 --consumer.config ${KAFKA_HOME}/bin/sasl-producer.properties > /dev/null &
+
+wait_for_port 8774
```

Comment:
Note to myself :slightly_smiling_face: consider separating it in several entrypoints for https://github.com/elastic/beats/pull/14087.
Don't do it here because it will only complicate this PR and it is not needed by now.